app/src/components/email.js

```diff
@@ -70,6 +84,18 @@ const email = {
     });
   },
 
+  /** Adds an email message to the queue
+   *  @param {object} message An email message object
+   *  @returns {object} A nodemailer result object
+   */
+  queueMailSmtp: (message, opts = {}) => {
+    const queue = require('./queue');
```

Comment:
not a fan of having email use queue and queue use email.  we should refactor this, and decouple.

Reply:
Refactored out the cyclic dependency